manager/dispatcher/dispatcher.go

```diff
@@ -1363,6 +1372,8 @@ func (d *Dispatcher) Session(r *api.SessionRequest, stream api.Dispatcher_Sessio
 	defer mgrCancel()
 	keyMgrUpdates, keyMgrCancel := d.keyMgrQueue.Watch()
 	defer keyMgrCancel()
+	rootCAUpdates, rootCACancel := d.rootCAQueue.Watch()
+	defer rootCACancel()
```

Comment:
The dispatcher changes look fine overall, but I'm wondering if we might combine these three channels into one. Each queue spawns its own goroutine. We could send a structure over the single channel that has fields for managers, the node object, and the root CA. Having such a structure would also make it clearer what types are involved.